Data/Yaml/Internal.hs

```diff
@@ -66,36 +66,56 @@ instance Exception ParseException where
 --   Instead of displaying the data constructors applied to their arguments,
 --   a more textual output is returned. For example, instead of printing:
 --
--- > AesonException "The key \"foo\" was not found"
+-- > InvalidYaml (Just (YamlParseException {yamlProblem = "did not find expected ',' or '}'", yamlContext = "while parsing a flow mapping", yamlProblemMark = YamlMark {yamlIndex = 42, yamlLine = 2, yamlColumn = 12}})))
 --
 --   It looks more pleasant to print:
 --
--- > Aeson exception: The key "foo" was not found
+-- > YAML parse exception at line 2, column 12,
+-- > while parsing a flow mapping:
+-- > did not find expected ',' or '}'
 --
 -- Since 0.8.11
 prettyPrintParseException :: ParseException -> String
-prettyPrintParseException NonScalarKey = "Non scalar key"
```

Comment:
Is there an actual change in behavior here, or is it just restructuring the code?